data/types/base/selfmonitored/self_monitored.go

```diff
@@ -9,8 +9,9 @@ import (
 type BloodGlucose struct {
 	base.Base `bson:",inline"`
 
-	Value *float64 `json:"value,omitempty" bson:"value,omitempty"`
-	Units *string  `json:"units,omitempty" bson:"units,omitempty"`
+	Value   *float64 `json:"value,omitempty" bson:"value,omitempty"`
+	Units   *string  `json:"units,omitempty" bson:"units,omitempty"`
+	SubType *string  `json:"subType,omitempty" bson:"subType,omitempty"`
```

Comment:
i think it is actually more the overloaded use of `subType` ... or thats how I read it. Other subtypes create different types of records with different fields, this one does not. http://developer.tidepool.io/data-model/device-data/types/smbg.html#subtype